<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameSetup</class>
 <widget class="QWidget" name="GameSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>383</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>551</width>
    <height>383</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="colorLabelCell">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>aa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Game rules</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QSlider" name="sliderB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>20</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Cell</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Shape</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>R      G      B</string>
   </property>
  </widget>
  <widget class="QSlider" name="sliderB_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>20</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Color, shape, size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>R      G      B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Cell birth</string>
   </property>
  </widget>
  <widget class="QSlider" name="sliderR_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>16</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Background</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>41</width>
     <height>266</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="box0R">
      <property name="text">
       <string notr="true">0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box1R">
      <property name="text">
       <string notr="true">1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box2R">
      <property name="text">
       <string notr="true">2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box3R">
      <property name="text">
       <string notr="true">3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box4R">
      <property name="text">
       <string notr="true">4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box5R">
      <property name="text">
       <string notr="true">5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box6R">
      <property name="text">
       <string notr="true">6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box7R">
      <property name="text">
       <string notr="true">7</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box8R">
      <property name="text">
       <string notr="true">8</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box9R">
      <property name="text">
       <string notr="true">9</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineX"/>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineY"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>41</width>
     <height>266</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="box0A">
      <property name="text">
       <string notr="true">0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box1A">
      <property name="text">
       <string notr="true">1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box2A">
      <property name="text">
       <string notr="true">2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box3A">
      <property name="text">
       <string notr="true">3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box4A">
      <property name="text">
       <string notr="true">4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box5A">
      <property name="text">
       <string notr="true">5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box6A">
      <property name="text">
       <string notr="true">6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box7A">
      <property name="text">
       <string notr="true">7</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box8A">
      <property name="text">
       <string notr="true">8</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="box9A">
      <property name="text">
       <string notr="true">9</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="boxShapes">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="sliderR">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>16</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="sliderG">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>16</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stay alive</string>
   </property>
  </widget>
  <widget class="QSlider" name="sliderG_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>16</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="colorLabelBkg">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>aa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cells number</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
